### ISSUE TITLE
Fix Drush Deploy Bootstrap Issue

### ISSUE THREAD

**Issue by jefflbrauer** [department-of-veterans-affairs/va.gov-cms#4812]
#4702 follow up

## AC
- [x] deploys don't fail because of Drush bootstrap error

**Comment by ElijahLynn** [comment]
I am working on this here > https://github.com/drush-ops/drush/issues/4702

**Comment by ElijahLynn** [comment]
Created a PR here > https://github.com/department-of-veterans-affairs/devops/pull/8856. Copied issue description from PR:

> This PR re-introduces a drush status command to potentially workaround an intermittent issue we were seeing that caused a bootstrap failure in drush deploy the step that now will run right after drush status. The idea is that the second time drush runs it does not have the issue, so drush status will absorb/workaround the issue.

I was not able to truly test this as the issue does not happen anymore and is not easily reproducible.

We used to have drush status in this same spot about a week or so ago, but when we upgraded to drush 10 it was removed because I didn't think it was needed anymore. So I think the might have been masked the whole time, especially since we don't check the drush status register for errors, nor do we want to now if we want to mitigate this issue.
